wellregistry/registry/admin/monitoring_location.py

```diff
@@ -34,15 +185,29 @@ class MonitoringLocationAdminForm(ModelForm):
     """
     Registry admin form.
     """
-
-    # pylint: disable=E1101
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super(MonitoringLocationAdminForm, self).__init__(*args, **kwargs)
```

Comment:
See Codacy. Please restore this. We should be using Python3 style super.

Reply:
fixed